cumulusci/tasks/salesforce/users/tests/test_permsets.py

```diff
@@ -199,3 +203,456 @@ def test_create_permset_raises(self):
 
         with pytest.raises(CumulusCIException):
             task()
+
+
+class TestCreatePermissionSetLicense:
+    @responses.activate
+    def test_create_permsetlicense(self):
+        task = create_task(
+            AssignPermissionSetLicenses,
+            {
+                "api_names": "PermSetLicense1,PermSetLicense2",
+            },
+        )
+
+        responses.add(
+            method="GET",
+            url=f"{task.org_config.instance_url}/services/data/v50.0/query/?q=SELECT+Id%2C%0A++++++++++++++++++++++++++++%28SELECT+PermissionSetLicenseId%0A+++++++++++++++++++++++++++++FROM+PermissionSetLicenseAssignments%29%0A++++++++++++++++++++++++FROM+User%0A++++++++++++++++++++++++WHERE+Username+%3D+%27test-cci%40example.com%27",
```

Comment:
This is just cosmetic but it might be good to write the query in a way that avoids including all these extra spaces. In Python one way to do this is:

    sf.query(
        "SELECT something "
        "from object"
    )

and those strings will be automatically concatenated into a single literal by the compiler.

Reply:
This is fixed.   The tests are much easier to digest.